supplemental/arithmetic/multiply.md

```diff
@@ -0,0 +1,93 @@
+# Multiply
+
+## Options
+
+### Overflow
+
+Multiplying two integers can trigger an overflow when the result is outside the
+representable range of the type class. This option controls what happens when
+this overflow occurs.
+
+#### SILENT
+
+If an overflow occurs then an integer value will be returned. The value is
+undefined. It may be any integer and can change from engine to engine or
+even from row to row within the same query.  The only constraint is that it
+must be a valid value for the result type class (e.g. multiplying two int16
+cannot yield an int32 on overflow)
+
+#### SATURATE
+
+If an overflow occurs then the largest (for positive overflow) or smallest
+(for negative overflow) possible value for the type class will be returned.
+
+#### ERROR
+
+If an overflow occurs then an error should be raised.
+
+### Rounding
+
+Multiplying two floating point numbers can yield a result that is not exactly
+representable in the given type class. In this case the value will be rounded.
+Rounding behaviors are defined as part of the IEEE 754 standard.
+
+#### TIE_TO_EVEN
+
+Round to the nearest value. If the number is exactly halfway between two
+values then round to the number whose least significant digit is even. Or,
+because we are working with binary digits, round to the number whose last digit
+is 0. This is the default behavior in many systems because it helps to avoid
+bias in rounding.
+
+#### TIE_AWAY_FROM_ZERO
```

Comment:
Ideally we would point to these common options instead of duplicating them everywhere.

Reply:
@sanjibansg let's use this PR to address the duplication issues

Reply:
This will have some framework-level changes. I have thought to make a separate YAML file containing the definition of such options. Functions that follow such options, and do not have any exceptions, should fetch the definition from that YAML, or should have their own definition here. I will make a separate PR for that.

Reply:
https://github.com/voltrondata/bft/pull/47